tests/src/JIT/Performance/RunBenchmarks/RunBenchmarks.cmd

```diff
@@ -0,0 +1,38 @@
+@setlocal
```

Comment:
I would add RunBenchmarks.cmd to the root of the test repo (or at least a thunk script) so that it is more discoverable.